manifests/s/Scopely/StarTrekFleetCommand/97742/Scopely.StarTrekFleetCommand.locale.en-US.yaml

```
@@ -0,0 +1,39 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.7-3-0-preview-8.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Scopely.StarTrekFleetCommand
+PackageVersion: "97742"
+PackageLocale: en-US
+Publisher: Xsolla
+PublisherUrl: https://scopely.com/
+PublisherSupportUrl: https://scopely.helpshift.com/a/star-trek-fleet-command/
+PrivacyUrl: https://scopely.com/privacy/
+# Author:
+PackageName: Star Trek Fleet Command
+PackageUrl: https://www.startrekfleetcommand.com/
+License: Proprietary
+LicenseUrl: https://scopely.com/tos/
+Copyright: Copyright (c) Scopely
+# CopyrightUrl:
+ShortDescription: Customize your fleet and crew to dominate the galaxy in this award-winning Star Trek 4x MMO.
+Description: |
```

```suggestion
Description: |-
```
The `-` provides cleaner output